.github/workflows/slovensko_digital_ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Slovensko.Digital CI
+
+on:
+  push:
+    branches: '**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      PGHOST: localhost
+      RAILS_ENV: test
+
+    services:
+      postgres:
+        image: postgres:12-alpine
+        env:
+          POSTGRES_USER: runner
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - run: yarn install
+      - run: bundle exec rails db:create db:structure:load --trace
+
+      - name: Setup Code Climate test-reporter
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./test-reporter
+          chmod +x ./test-reporter
+          ./test-reporter before-build
```

Comment:
nevieme tieto commandy spravit cez action? pozerala si to? https://github.com/marketplace?query=code+climate&type=actions

Reply:
@pavolzbell hento vyzera na nejake moc nepouzivane veci a ten pouzivany je na NPM, cize sa skor priklanam k tomuto priamemu rieseniu. 

@luciajanikova ale niekde tu bude treba nejaky test reporter id alebo cosi, inak to nebude vediet asi reportovat

Reply:
@jsuchal NPM s tym podla mna nic nema a s touto action https://github.com/marketplace/actions/code-climate-coverage-action to vieme pekne nahradit takto:
```
steps:
  - uses: paambaati/codeclimate-action@v2.7.5
    env:
      CC_TEST_REPORTER_ID: ...
    with:
      coverageCommand: bundle exec rspec
```

cc @luciajanikova

Reply:
@pavolzbell Vďaka, idem to skúsiť spustiť s tou action